packages/default/scss/menu/_layout.scss

```diff
@@ -35,6 +34,10 @@
         flex: none;
         position: relative;
         user-select: none;
+
+        &.k-selected {
+            font-weight: var( --kendo-menu-item-selected-font-weight, #{$kendo-menu-item-selected-font-weight} );
```

Comment:
The font weight here should be set in the following way:
font-weight: $kendo-menu-item-selected-font-weight;